src/main/java/com/internet/shop/Main.java

```diff
@@ -0,0 +1,54 @@
+package com.internet.shop;
+
+import com.internet.shop.dao.ProductDao;
+import com.internet.shop.dao.jdbc.ProductDaoJdbcImpl;
+import com.internet.shop.model.Product;
+import com.internet.shop.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.SQLException;
+
+public class Main {
+    public static void main(String[] args) {
+        ProductDao productDaoJdbc = new ProductDaoJdbcImpl();
```

Comment:
let's use injector

Reply:
![image](https://user-images.githubusercontent.com/34981418/93751493-d82bb300-fc05-11ea-910c-9140529a75a9.png)
![image](https://user-images.githubusercontent.com/34981418/93751614-03160700-fc06-11ea-8906-0e67f3d79636.png)

I have an exception when I use Injector

Reply:
UPD: solved after Q&A. Now I use Injector here and everything is fine